src/spring-cloud/azext_spring_cloud/_params.py

```diff
@@ -74,8 +74,11 @@ def load_arguments(self, _):
         c.argument('name', name_type, help='Name of app.')
 
     with self.argument_context('spring-cloud app create') as c:
-        c.argument(
-            'is_public', arg_type=get_three_state_flag(), help='If true, assign public domain', default=False)
+        c.argument('assign_endpoint', arg_type=get_three_state_flag(),
+                   help='If true, assign endpoint', default=False,
+                   options_list=['--assign-endpoint', c.deprecate(target='--is-public', redirect='--assign-endpoint')])
+        c.argument('is_public', arg_type=get_three_state_flag(),
```

Comment:
pls refine the deprecation message. such as suggest a new parameter or clarify on latest behavior

Reply:
Resolved. Please review again.